src/store/firestoreRequestEvaluations/selectors.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createSelector } from 'reselect';
+
+import { AppState } from '..';
+
+function getAllRequestEvaluationsFromState(state: AppState) {
+  return state.firestore.requests.evaluations;
+}
+
+function getSelectedRequestEvaluationFromState(
+  state: AppState,
+  selectedRequestEvaluationId: string
+) {
+  return state.firestore.requests.evaluations.find(
+    ({ requestId }) => requestId === selectedRequestEvaluationId
+  );
+}
+
+export const getAllRequestEvaluations = createSelector(
```

Comment:
What do these selectors do? They seem redundant (in this case just passing through `getAllRequestEvaluationsFromState`)